lnbits/core/crud/payments.py

```diff
@@ -268,13 +269,15 @@ async def delete_expired_invoices(
 async def create_payment(
     checking_id: str,
     data: CreatePayment,
+    created_at: int | None = None,
+    updated_at: int | None = None,
```

Comment:
those args should not be passed, those should be created automatically when the model is initialized.